src/Data/CallbackRequest/Pisa/pisaCallbackRequestDataClass.ts

```diff
@@ -1,16 +1,12 @@
 import { ClientError, provideDataClass } from 'scrivito'
 import { pisaClient } from '../../pisaClient'
-import {
-  DataClassAttributes,
-  DataConnection,
-  DataIndexResponse,
-  RawItem,
-} from '../../types'
+import { DataConnection, DataIndexResponse, RawItem } from '../../types'
+import { filterSchema } from '../../filterSchema'
 
-export function pisaCallbackRequestDataClass(attributes: DataClassAttributes) {
+export function pisaCallbackRequestDataClass() {
   // callback-request is more or less a "singleton". It only offers PUT, GET and DELETE.
   return provideDataClass('CallbackRequest', {
-    attributes,
+    attributes: () => filterSchema('callback-request'),
```

Comment:
* [ ] FYI I don't see why we need a function wrapper. How about `attributes: filterSchema('callback-request')`?

Reply:
Without the function wrapper `filterSchema` would be requested for all data classes during boot - regardless of them being used or not. With the function wrapper the attributes are only requested, if the data class is in use.